libs/postgres_backend/src/lib.rs

```diff
@@ -442,10 +442,20 @@ impl<IO: AsyncRead + AsyncWrite + Unpin> PostgresBackend<IO> {
             trace!("got message {:?}", msg);
 
             let result = self.process_message(handler, msg, &mut query_string).await;
-            self.flush().await?;
+            tokio::select!(
```

Comment:
@hlinnaka in this change, I'm also replacing the two flush calls (one before `match` and one within it) with a single call before the match.  The existing code reads to me like the second flush is a no-op, but pinging you in case there was something subtle going on here.

Reply:
I agree it looks like mistake